internal/backend/xattr_windows.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Francisco Souza. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// TODO: this package works around missing Windows support in xattr:
+// https://github.com/pkg/xattr/issues/47
+
+package backend
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+const XATTR_SUFFIX = ".metadata"
```

Comment:
Same as above regarding the naming convention
```suggestion
const xattrSuffix = ".metadata"
```

Reply:
Done.